engine/crates/engine-v2/src/operation/mod.rs

```diff
@@ -69,7 +71,7 @@ where
     }
 }
 
-#[derive(serde::Serialize, serde::Deserialize, IndexedFields)]
+#[derive(Clone, serde::Serialize, serde::Deserialize, IndexedFields)]
```

Comment:
Is cloning this really necessary? Smells like it's an expensive operation. Are we cloning it somewhere?